ffmpeg/api_test.go

```diff
@@ -1175,3 +1175,69 @@ func TestTranscoder_SkippedFrames(t *testing.T) {
   `
 	run(cmd)
 }
+
+func audioOnlySegment(t *testing.T, accel Acceleration) {
+	// Reproducing #203
+	run, dir := setupTest(t)
+	defer os.RemoveAll(dir)
+
+	cmd := `
+    # run segmenter and sanity check frame counts . Hardcode for now.
+    ffmpeg -loglevel warning -i "$1"/../transcoder/test.ts -c:a copy -c:v copy -f hls test.m3u8
+    ffprobe -loglevel warning -select_streams v -count_frames -show_streams test0.ts | grep nb_read_frames=120
+    ffprobe -loglevel warning -select_streams v -count_frames -show_streams test1.ts | grep nb_read_frames=120
+    ffprobe -loglevel warning -select_streams v -count_frames -show_streams test2.ts | grep nb_read_frames=120
+    ffprobe -loglevel warning -select_streams v -count_frames -show_streams test3.ts | grep nb_read_frames=120
+
+    # drop all video frames from seg2
+    ffmpeg -loglevel warning -i test2.ts -c:a copy -c:v libx264 -vf "fps=fps=1/100000" test02.ts
```

Comment:
Question for my own personal understanding:

Do we drop all video frames here by setting the output fps to 1/100000? I'm guessing that in this case the fps is so low that no video frames end up in the output?

Reply:
> Do we drop all video frames here by setting the output fps to 1/100000? I'm guessing that in this case the fps is so low that no video frames end up in the output?

Yes exactly! I first used the `-vn` parameter instead of FPS filter, but that dropped the video stream altogether, which did not match the scenario in the MistServer samples